python/pyspark/sql.py

```diff
@@ -2284,6 +2284,18 @@ def addColumn(self, colName, col):
         """
         return self.select('*', col.alias(colName))
 
+    def to_pandas(self):
+        """
+        Collect all the rows and return a `pandas.DataFrame`.
+
+        >>> df.to_pandas()  # doctest: +SKIP
```

Comment:
@rxin we have no pandas in jenkins now. We could enable this test once we have it.

@pwendell I think `pandas` is just the project name, `panda` is not meaningful in their context.